templates/write_code_headers.py.in

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+#
+# Scans a source tree for python files and writes the header template on top of each file
+
+import os
+import sys
+import optparse
+from datetime import datetime
+
+SRC_DIR = '${buildout:directory}/src/senaite/sync'
```

Comment:
@xispa @ramonski I wasn't able here to find out how to specify the source directory so I finally run the script by passing the route as a command line argument. Any ideas on how it should be specified?

Reply:
Ok, got it!

Reply:
Hi @juangallostra, can you please hook this script in the local `buidlout.cfg` like here: https://github.com/senaite/senaite.core/blob/master/buildout.cfg#L60